webapp/src/components/PollsPage/PollsPage.tsx

```diff
@@ -1,136 +1,125 @@
 import * as React from 'react'
-import { Link } from 'react-router-dom'
-import { Header, Loader, Table } from 'decentraland-ui'
-import { locations } from 'locations'
-import { PollsPageProps } from 'components/PollsPage/types'
-import { PollWithAssociations } from 'modules/poll/types'
-import { Wallet } from 'modules/wallet/types'
+import * as queryString from 'query-string'
+import { Loader } from 'decentraland-ui'
+import { Props } from './PollsPage.types'
 import { t } from '@dapps/modules/translation/utils'
-import { formatNumber } from '@dapps/lib/utils'
-import { isDistrictToken } from 'modules/token/district_token/utils'
-import { getBalanceInPoll } from 'modules/wallet/utils'
-import './PollsPage.css'
-import Token from 'components/Token'
+import PollsTable from './PollsTable'
 
-const sortByContributions = (wallet: Wallet) => (
-  pollA: PollWithAssociations,
-  pollB: PollWithAssociations
-) => {
-  const contribA = getBalanceInPoll(wallet, pollA) || 0
-  const contribB = getBalanceInPoll(wallet, pollB) || 0
-  if (contribA > contribB) {
-    return -1
-  } else if (contribB > contribA) {
-    return 1
-  }
-  return pollA.title > pollB.title ? 1 : -1
-}
+export default class PollsPage extends React.PureComponent<Props> {
+  activeRows: number = 10
+  expiredRows: number = 10
+  shouldFetchActivePolls: boolean = false
+  shouldFetchExpiredPolls: boolean = false
 
-export default class PollsPage extends React.PureComponent<PollsPageProps> {
   componentWillMount() {
-    this.props.onFetchPolls()
+    this.fetchActivePolls()
+    this.fetchExpiredPolls()
   }
 
-  render() {
-    const { polls, wallet, isLoading } = this.props
+  componentWillReceiveProps({ location: { search } }: Props) {
```

Comment:
_NO action required_

No matter how many times I see this syntax it rubs me the wrong way.

Reply:
haha, true, I can deconstruct `search` in the line below